segmenters/image/TorchObjectDetectionSegmenter/__init__.py

```diff
@@ -99,25 +110,45 @@ def segment(self, blob: 'np.ndarray', *args, **kwargs) -> List[Dict]:
         :param args:  Additional positional arguments
         :param kwargs: Additional keyword arguments
         """
-        raw_img = np.copy(blob)
+        raw_img = np.copy(blob) # (2, 681, 1264, 3) with imgs/cars.jpg
+        # "Ensure the color channel axis is the default axis." i.e. c comes first
+        # e.g. (h,w,c) -> (c,h,w) / (n,h,w,c) -> (c,n,h,w)
```

Comment:
color needs to come second if there is batching involved

Reply:
U mean as a convention ?